src/components/forms/ComboBox/ComboBox.stories.tsx

```diff
@@ -40,6 +40,8 @@ export const defaultComboBoxWithPropOptions = (): React.ReactElement => {
         name="input-ComboBox"
         options={fruitList}
         onChange={noop}
+        inputProps={{ onChange: (e) => console.log('input') }}
```

Comment:
@suzubara am I understanding correctly that we want to be able to pass an "onChange" to the input like this, and we need to not only make sure it triggers on select/reset/mount, but also override the default behavior of triggering on when the user types within the input?

I don't see where USWDS overrides the "typing" change events. Does a JSX/React created `input` element have a different default "onChange" behavior?